Lib/test/test_argparse.py

```diff
@@ -3764,6 +3764,28 @@ class TestHelpUsage(HelpTestCase):
     version = ''
 
 
+class TestHelpUsageWithParentheses(HelpTestCase):
```

Comment:
I don't see any tests that actually add parentheses for a group with more than one action. Could you add one?

Reply:
That's a good idea. But I think the testcase you want already exists using the code below. Please review it.

https://github.com/python/cpython/blob/66aa78cbe604a7c5731f074b869f92174a8e3b64/Lib/test/test_argparse.py#L2777

Reply:
You're right, thanks! There are a lot of tests, I must have glanced over that one.